python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -41,7 +42,7 @@ def _getName(obj):
 
 _imported_GangaList = None
 
-do_not_copy = ['_index_cache', '_parent', '_registry', '_data', '_read_lock', '_write_lock', '_proxyObject']
+do_not_copy = ['_index_cache', '_index_cache_dict', '_parent', '_registry', '_data', '_data_dict', '_read_lock', '_write_lock', '_proxyObject']
```

Comment:
`_index_cache` is the descriptor and `_index_cache_dict` is the actual Python dict underlying it. Similarly for the `_data`.

The usual pattern here is for publicly accessible object, you would have an attribute called `foo` with some underlying data, perhaps stored in a class variable called `_foo` which would be accessed by:
```python
@property
def foo(self):
    return self._foo
```
Since `_data` and `_index_cache` are already semi-internal, a different name was needed.

I'm not sure that these are all needed in the `do_not_copy` though and in fact I'd very much like to get rid of it sometime soon.